Failed to stop and delete the process during download.(Error when video finished downloading)

### Steps to reproduce

A) 1.Error when download stopped.
2.The video continues to download even if you delete the download.
B)Error when video finished downloading

### Expected behavior

Stopping and deleting the video should be successful

### Actual behavior

java.lang.RuntimeException: Error receiving broadcast Intent { act=com.github.libretube.services.DownloadService.ACTION_SERVICE_STOPPED flg=0x10 } in com.github.libretube.receivers.DownloadReceiver@c02609e
	at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$getRunnable$0$LoadedApk$ReceiverDispatcher$Args(LoadedApk.java:1816)
	at android.app.LoadedApk$ReceiverDispatcher$Args$$ExternalSyntheticLambda0.run(Unknown Source:2)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8669)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
Caused by: android.content.ActivityNotFoundException: Unable to find explicit activity class {com.github.libretube/com.github.libretube.ui.activities.MainActivity}; have you declared this activity in your AndroidManifest.xml?
	at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:2098)
	at android.app.Instrumentation.execStartActivity(Instrumentation.java:1747)
	at android.app.Activity.startActivityForResult(Activity.java:5473)
	at androidx.activity.ComponentActivity.startActivityForResult(ComponentActivity.java:2)
	at android.app.Activity.startActivityForResult(Activity.java:5431)
	at androidx.activity.ComponentActivity.startActivityForResult(ComponentActivity.java:1)
	at android.app.Activity.startActivity(Activity.java:5817)
	at android.app.Activity.startActivity(Activity.java:5770)
	at com.github.libretube.receivers.DownloadReceiver.onReceive(DownloadReceiver.kt:64)
	at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$getRunnable$0$LoadedApk$ReceiverDispatcher$Args(LoadedApk.java:1806)
	... 9 more
    

### LibreTube version

0.12.0

### Android version

Android 12

### Other details


https://user-images.githubusercontent.com/124310688/219204110-3d86e2a2-521c-48a2-b97f-5e26881e4db9.mp4



### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -80,19 +80,30 @@
             android:screenOrientation="user"
             android:supportsPictureInPicture="true"
             android:windowSoftInputMode="adjustPan">
-            <intent-filter>
-                <action android:name="android.intent.action.MAIN" />
-
-                <category android:name="android.intent.category.LAUNCHER" />
-                <category android:name="android.intent.category.LEANBACK_LAUNCHER" />
-            </intent-filter>
-
             <!-- allow being recognized as a music player -->
             <intent-filter>
                 <action android:name="android.intent.action.MUSIC_PLAYER" />
                 <category android:name="android.intent.category.CATEGORY_APP_MUSIC" />
             </intent-filter>
         </activity>
+
+        <activity-alias
+            android:name=".Default"
+            android:configChanges="screenSize|smallestScreenSize|screenLayout|orientation"
+            android:enabled="true"
+            android:exported="true"
+            android:icon="@mipmap/ic_launcher"
+            android:roundIcon="@mipmap/ic_launcher_round"
+            android:supportsPictureInPicture="true"
+            android:targetActivity=".ui.activities.MainActivity"
+            android:windowSoftInputMode="adjustPan" >
+        <intent-filter>
+            <action android:name="android.intent.action.MAIN" />
+
+            <category android:name="android.intent.category.LAUNCHER" />
+            <category android:name="android.intent.category.LEANBACK_LAUNCHER" />
+        </intent-filter>
+        </activity-alias>
 
         <activity-alias
             android:name=".IconGradient"

--- a/app/src/main/java/com/github/libretube/ui/adapters/IconsSheetAdapter.kt
+++ b/app/src/main/java/com/github/libretube/ui/adapters/IconsSheetAdapter.kt
@@ -41,7 +41,7 @@ class IconsSheetAdapter : RecyclerView.Adapter<IconsSheetViewHolder>() {
             val activityAlias: String,
         ) {
             object Default :
-                AppIcon(R.string.defaultIcon, R.mipmap.ic_launcher, "ui.activities.MainActivity")
+                AppIcon(R.string.defaultIcon, R.mipmap.ic_launcher, "Default")
 
             object DefaultLight :
                 AppIcon(R.string.defaultIconLight, R.mipmap.ic_launcher_light, "DefaultLight")

